docs/users.rst

```diff
@@ -900,30 +900,37 @@ This example uses RHEL 8.4 EUS's Pulp repository IDs:
   - rhel-8-for-s390x-appstream-eus-rpms__8_DOT_4
 
 
-Using Artifacts from Koji
+Using Artifacts from Koji or Project Newcastle
 -------------------------
 
 During a container build, it might be desirable to fetch some artifacts
-from an existing Koji build. For instance, when building a Java-based
-container, JAR archives from a Koji build are required to be added to
-the resulting container image.
+from an existing Koji build or a Project Newcastle build. For instance, when building
+a Java-based container, JAR archives from a Koji build or PNC build are required
+to be added to the resulting container image.
 
 
 The atomic-reactor pre-build plugin, fetch_maven_artifacts, can be used
 for including non-RPM content in a container image during build time.
-This plugin will look for the existence of two files in the git repository
+This plugin will look for the existence of three files in the git repository
 in the same directory as the Dockerfile:
-fetch-artifacts-koji.yaml and fetch-artifacts-url.yaml.  (See `fetch-artifacts-url.json`_ and `fetch-artifacts-nvr.json`_ for their YAML schema.)
-
-.. _`fetch-artifacts-url.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/fetch-artifacts-url.json
+fetch-artifacts-koji.yaml, fetch-artifacts-pnc.yaml and fetch-artifacts-url.yaml.
+(See `fetch-artifacts-nvr.json`_, `fetch-artifacts-pnc.json`_ and
+`fetch-artifacts-url.json`_ for their YAML schema.)
 
 .. _`fetch-artifacts-nvr.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/fetch-artifacts-nvr.json
 
-The first is meant to fetch artifacts from an existing Koji build.  The second
+.. _`fetch-artifacts-pnc.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/fetch-artifacts-pnc.json
+
+.. _`fetch-artifacts-url.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/fetch-artifacts-url.json
+
+The first is meant to fetch artifacts from an existing Koji build. The second is
```

Comment:
can we rephrase this whole block?

instead of saying first,second,third, it would be nicer just write that yaml filename

about those combinations, I'd just simply say that, all those files can exist but aren't mandatory

and remote that part about order in which they will be processed

Reply:
@MartinBasti

Reply:
Yes, make future life easier :D No hardcoded numbers

Reply:
Instead of deletion, I'd just make it as list `files will be processed in this order: <1>, <2>, <3>`

Reply:
updated.